src/crawler/index.js

```diff
@@ -2,8 +2,12 @@ const fs = require('fs');
 const { join, sep } = require('path');
 
 function getDetails(data) {
-	const matadata = data.match(/---(.*(\r)?\n)*---/g)[0];
-	const details =  matadata.match(/(.*):(.*)/g).reduce((obj, detail) => {
+	const markDown = data.match(/---(.*(\r)?\n)*---/g);
+	if (!markDown) {
+		return JSON.parse(data);
```

Comment:
can we wrap this in `try, catch`. So if a third format comes this doesnt blow up